code/src/containers/best-sellers/BestSellers.test.tsx

```diff
@@ -55,7 +55,8 @@ describe("BestSellers", () => {
     expect(useGetUserProductsQuery).toHaveBeenCalledWith({
       page: 0,
       size: 5,
-      lang: "en"
+      lang: "en",
+      tags: ""
```

Comment:
Why this is empty here? I suggest making `tags` optional property

Reply:
Done